website/src/content/docs/docs/emitters/sdk-clients/introduction.md

```diff
@@ -0,0 +1,134 @@
+| @typespec/http-client-java   | Java         | `0.1.9`                  | `emitter-output-dir`     |
+| @typespec/http-client-csharp | .NET         | `0.1.9-alpha.20250113.2` | `emitter-output-dir`     |
+
+### Common Configuration Options
+
+- `emitter-output-dir`: Defines where the generated SDK files will be stored.
+
+Below is an example of the `package.json` snippet where client emitters are defined:
+
+```json
+  "dependencies": {
+    "@typespec/http-client-csharp": "^0.1.9-alpha.20250113.2",
+    "@typespec/http-client-java": "^0.1.9",
+    "@typespec/http-client-python": "^0.6.6",
+    "@azure-tools/typespec-ts": "^0.38.1",
+  }
+```
+
+#### Note: Check for the Latest Version
+
+To ensure you are using the latest version of the packages, visit [npmjs](https://www.npmjs.com/) and search for the relevant packages.
+
+## Language-Specific Settings
+
+### JavaScript Client Emitter
+
+Generally no additional setting is required for JavaScript. However it's recommended to specify `packageDetails` which would provide package metadata in `package.json` and `README.md` files.
+
+#### packageDetails
+
+Provide the metadata for `package.json`, `README.md` information.
+
+| Property    | Description                                                            |
+| ----------- | ---------------------------------------------------------------------- |
+| name        | Package name used in `package.json`                                    |
+| description | Package description used in `package.json` file                        |
+| version     | Detailed version for your package, the default value is `1.0.0-beta.1` |
+
+Example configuration:
+
+```yaml
+packageDetails:
+  name: "${your_package_name}"
+  version: 1.0.0
+```
+
+### Python Client Emitter
+
+No additional setting needed for Python.
+
+### Java Client Emitter
+
+#### Required Dependencies
+
+Java (Java Development Kit) and Maven (Apache Maven) is required to be installed, before running Java client emitter.
+
+The required version and the URL for downloading is specified below:
+
+- [Java](https://docs.microsoft.com/java/openjdk/download) 17 or above. (Verify by running `java --version`)
+- [Maven](https://maven.apache.org/download.cgi). (Verify by running `mvn --version`)
+
+#### Configuration Options for Java
+
+No additional setting needed for Java.
+
+### .NET Client Emitter
+
+## Running Language-Specific Emitters in CLI
+
+1. Ensure that your package.json file is correctly configured to include the necessary dependencies for running the emitters
+
+1. Update the tspconfig.yaml file for properly configured for the language-specific emitter.
+
+```yaml
+emit:
+  - "@typespec/http-client-csharp"
+  - "@typespec/http-client-java"
+  - "@typespec/http-client-python"
+  - "@azure-tools/typespec-ts"
+options:
+  "@typespec/http-client-csharp":
+    emitter-output-dir: "{project-root}/../clients/dotnet"
```

Comment:
why are the path going `..`, this make some assumption that they want to add it outside of their project which I would think is not what someone would epxect when getting started(start poluting some other folder)

Reply:
Removed  `..`